galera/Project1/Developer.h

```diff
@@ -10,6 +10,8 @@ class Developer : public Employee
     string _language;
 
 public:
-    void setLanguage(string language);
+    enum class  language { c, c_sharp, python, c_plusplus, java };
```

Comment:
Enum should be named as type, that is `Language`. Also, be consistent with a style and name everyting `CSharp`, `CPlusPlus` etc.